### PR TITLE
The gas_price value is 0 on privateNet

### PR DESCRIPTION
**Pull request by yangbinnnn**
Ignore the gas_price that user config on privateNet

### CODE REVIEW

File: punica/deploy/deploy_contract.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 class Deploy:
     @staticmethod
     def generate_signed_deploy_transaction(hex_avm_code: str, project_path: str = '', wallet_file_name: str = '',
```
Comment (lucas7788) [inline punica/deploy/deploy_contract.py]: the gasprice can be 500 or 0 in private Net

**Comment by lucas7788** [comment]
@yangbinnnn   the gasprice can be 500 or 0 in private Net

**Comment by yangbinnnn** [comment]
In my private net, only one accout hold all the ONT, But gas_price = 500 has this problem
```
(59000, 'Other Error, transactor 32e77d475062f9edbd5f2a8e4ef4f067d90cf4fc has no balance enough to cover gas cost 10200000000')
```

Ignore the gas_price = 500, fixed this problem

Sorry, i'm new to ONT, there could be another reasons

**Comment by lucas7788** [comment]
@yangbinnnn  send transaction need cost ong not ont, so you should make sure you have enough ong

**Comment by yangbinnnn** [comment]
No gas in my private net

**Pull request closed without merging**